models/forms.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * name : forms.js.
+ * author : Prajwal.
+ * created-date : 08-May-2024.
+ * Description : Schema for Forms.
+ */
+
+const { Schema } = require('mongoose')
+
+module.exports = {
+	name: 'forms',
+	schema: {
+		type: {
+			type: String,
+			required: true,
+			unique: true,
+			index: true,
+		},
+		subType: {
+			type: String,
+			required: true,
+		},
+		version: {
```

Comment:
@Prajwal17Tunerlabs version has to increment when we update a form, is it covered?

Reply:
added pre-hook to increment the version by 1 whenever updation of the form takes place